src/views/IDUploader/index.js

```diff
@@ -16,7 +16,7 @@ import IdCardGreen from '../IDUploader/image-green.png';
 import IdCardTop from '../IDUploader/image-top.png';
 import Cropper from 'react-cropper';
 import 'cropperjs/dist/cropper.css';
-import './IDUploader.css';
+import './IDUploader.scss';
```

Comment:
I don't think this was supposed to change. Can we change it back?

Reply:
Change made